projects/plugins/jetpack/tests/e2e/specs/connection.test.js

```diff
@@ -15,33 +20,49 @@ process.env.SKIP_CONNECT = true;
  * @group connection
  */
 describe( 'Connection', () => {
-	beforeAll( async () => {
-		await execMultipleWpCommands(
-			'wp option delete e2e_jetpack_plan_data',
-			'wp option delete jetpack_active_plan',
-			'wp option delete jetpack_private_options',
-			'wp option delete jetpack_sync_error_idc'
-		);
-		await page.reload();
-		await page.reload();
+	afterEach( async () => {
+		await execShellCommand( 'yarn tunnel-reset' );
+		global.siteUrl = resolveSiteUrl();
+		await resetWordpressInstall();
+		await loginToWpComIfNeeded( 'defaultUser', true );
+		await loginToWpSite( true );
 	} );
 
-	beforeEach( async () => {
-		await DashboardPage.visit( page );
+	it( 'In-place', async () => {
+		await step( 'Can start in-place connection', async () => {
+			await DashboardPage.visit( page );
```

Comment:
Dashboard -> Jetpack navigation is identical, maybe it can be extracted in `beforeEach`

Reply:
Almost. userless flow has a logout step before the above navigation. If I'll figure out how to do that, I'll extract it.